testing/e2e/clients/aks.go

```diff
@@ -133,7 +133,7 @@ func NewAks(ctx context.Context, subscriptionId, resourceGroup, name, location s
 				{
 					Name:   to.Ptr("default"),
 					VMSize: to.Ptr("Standard_DS3_v2"),
-					Count:  to.Ptr(int32(3)),
+					Count:  to.Ptr(int32(6)),
```

Comment:
this is a huge jump in requirements. have we tried 4 and 5? doesn't seem like a default backend client server test should warrant this much of a bump

Reply:
Got a successful run with 4!